### DISCUSSION TITLE
疑似缺包

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4598] [Troubleshooting | 故障排查]
终端报这个错误：

meson.build:1902:2: ERROR: Invalid version of dependency, need 'wayland-protocols' ['>= 1.24'] found '1.17'

怎么将这个wayland-protocols升级到最高，或者说去哪里找到mesa23.1最新驱动的deb包，我的目的就是想安装最新的mesa驱动https://bbs.deepin.org/phone/zh/post/257513

**Comment by Decodetalkers** [comment]
Mesa要我们打包，你不要乱动
